Directory.Build.props

```diff
@@ -85,7 +85,38 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(RepositoryBranch)' == ''">
```

Comment:
This is a lot of code and I don't really it's worth adding it, especially given the fix is already pending in the official package. I'd rather just wait for that to be available, and then we can just update this PR to bump the dependency 🙂

Reply:
Yeah, that might be best. Also, if you only build on GH actions, might be worth keeping just that single part?

Reply:
Maybe, but given you've already opened [a PR in ther](https://github.com/dotnet/reproducible-builds/pull/27) and it also already has an approval, I'd rather just wait for that one to be in so we can do this properly from the start. I can leave this open if you want, so you can then just rebase and bump that dependency 🙂